### PR TITLE
Chrome 124 supports read-write and read-only storageTexture.access + Chrome Android 121 supports WGSLLanguageFeatures

### PR DESCRIPTION
**Pull request by chrisdavidmills**
#### Summary

Chrome 124 supports the `readonly_and_readwrite_storage_textures` language extension (see [`WGSLLanguageFeatures`](https://developer.mozilla.org/en-US/docs/Web/API/WGSLLanguageFeatures)), which allows the `"read-write"` and `"read-only"` [`storageTexture.access`](https://developer.mozilla.org/en-US/docs/Web/API/GPUDevice/createBindGroupLayout#access) types to be set when specifying `storageTexture` bind group entry types in a bind group layout.

This PR adds data points for both the language extension and the access types.

See https://developer.chrome.com/blog/new-in-webgpu-124#read-only_and_read-write_storage_textures for the data source.

#### Test results and supporting details

Spec PR: https://github.com/gpuweb/gpuweb/pull/4326

#### Related issues

Project issue: https://github.com/mdn/content/issues/36346.

### CODE REVIEW

File: api/WGSLLanguageFeatures.json [hunk 1]
```diff
@@ -79,6 +79,46 @@
...
+          "spec_url": "https://gpuweb.github.io/gpuweb/wgsl/#language_extension-readonly_and_readwrite_storage_textures",
+          "tags": [
+            "web-features:webgpu"
+          ],
+          "support": {
+            "chrome": {
+              "version_added": "124"
+            },
+            "chrome_android": {
+              "version_added": false
```
[the remark this answers is not in the document]
Comment (chrisdavidmills) [reply api/WGSLLanguageFeatures.json]: Done, in my last commit.

File: api/GPUDevice.json [hunk 2]
```diff
@@ -177,6 +177,49 @@
...
+            "spec_url": "https://gpuweb.github.io/gpuweb/wgsl/#memory-access-mode",
+            "tags": [
+              "web-features:webgpu"
+            ],
+            "support": {
+              "chrome": {
+                "version_added": "124"
+              },
+              "chrome_android": {
+                "version_added": false
```
[the remark this answers is not in the document]
Comment (chrisdavidmills) [reply api/GPUDevice.json]: Done, in my last commit

**Comment by caugner** [comment]
Apologies @chrisdavidmills, something went wrong when I tried to resolve the merge conflict.

Could you please `git reset --hard e42049d994e031fe8a380fd81ce29ff36ea7d24d && git push --force` to undo my mischief?

**Review by caugner — COMMENTED** [review]
FWIW I just added mirror support to my compact diff, and here's the output for this PR: 

image

It looks like this PR not only adds data for read-write and read-only storageTexture.access, but also updates some versions for `api.WGSLLanguageFeatures`. Is this intentional, i.e. are these changes directly related?

**Comment by chrisdavidmills** [comment]
> git reset --hard e42049d994e031fe8a380fd81ce29ff36ea7d24d && git push --force

@caugner  done!

I'll have a look at the conflict now.

**Comment by chrisdavidmills** [comment]
@caugner ok, conflict fixed

**Comment by chrisdavidmills** [comment]
It was intentional to update some of the values for `api.WGSLLanguageFeatures` while doing this — they were erroneously marked as wrong for the longest time.

**Review by caugner — APPROVED** [review]
LGTM, thanks!

Compact diff grouped by ~~value~~ change:

image

**Pull request merged**